### PR TITLE
Fixed problem with dumping broken names breaks workflow

### PR DESCRIPTION
**Pull request by pazembrz**

### CODE REVIEW

File: tests/test_name.py [hunk 1]
```diff
@@ -420,6 +420,17 @@ def test_parsed_name_initials():
     assert expected == parsed_name.first_initials_list
 
 
+def test_parsed_wrong_names_and_not_fail():
+    names = [('Proffesor.M.', 'Proffesor.M.'), ('ˇ Sirˇ', u'Sirˇ, ˇ.'),
+             ('Belvedere, G., Ed', 'Belvedere, G., Ed.'),
+             ('O. Madelung, M. Schulz, H. Weiss, Editor.',
+              'O. Madelung, M. Schulz, H. Weiss, Editor.')
```
[the remark this answers is not in the document]
Comment (pazembrz) [reply tests/test_name.py]: Yes these are the real cases found in Sentry by Harris.

[the remark this answers is not in the document]
Comment (michamos) [reply tests/test_name.py]: That should not happen, they should be split up by the other function I mentioned. @drjova are you sure?

[the remark this answers is not in the document]
Comment (drjova) [reply tests/test_name.py]: Those are real examples found in Sentry <https://github.com/inspirehep/inspire-schemas/pull/348>

File: tests/test_name.py [hunk 2]
```diff
@@ -420,6 +421,18 @@ def test_parsed_name_initials():
     assert expected == parsed_name.first_initials_list
 
 
+def test_parsed_wrong_names_and_not_fail():
+    names = [('Proffesor.M.', 'Proffesor.M.'),
+             ('ˇ Sirˇ', u'Sirˇ, ˇ.'),
```
Comment (drjova) [inline tests/test_name.py]: why this has ``u`` and the other don't? consistency :)

Comment (pazembrz) [reply tests/test_name.py]: I added `u` because it outputs utf string, but I can change everything to utf strings

**Pull request merged**